main.go

```diff
@@ -201,11 +193,27 @@ func main() {
 		return jsonFormattedData, nil
 	}
 
-	wesRequestWithDetailsAndPublic := func(c echo.Context) error {
-		qs := url.Values{}
-		qs.Add("with_details", "true")
-		qs.Add("public", "true")
-		return wesRequest("/runs", qs, c, jsonDeserialize)
+	gohanRequestPublic := func(c echo.Context) error {
```

Comment:
you could make this a generic function and pass in the gohan/katsu URL as an argument, since the logic and structure should be identical for both endpoints.